CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.2.0
+
+* Correct bounds for base. GHC support for versions older than 8.0 was dropped.
```

Comment:
To follow convention from the previous changes this ideally should be similar to
```md
## 0.4.2.0

### Removed
*  GHC support for versions older than 8.0. Bounds for base corrected
```